.circleci/config.yml

```diff
@@ -6,18 +6,29 @@ jobs:
       - image: "cimg/elixir:1.13.3"
     steps:
       - checkout
+      - run: mix local.hex --force
+      - run: mix deps.get
```

Comment:
I wonder if you can run `mix.deps.get` once, cache the dependencies and have other jobs use the cache instead?